script/smoke_test.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash -e
+
+# This is an integration test to generally check the quality of parsing.
+
+SCALA_SCALA_LIBRARY_EXPECTED=87
+SCALA_SCALA_COMPILER_EXPECTED=46
+DOTTY_COMPILER_EXPECTED=35
+
+if [ ! -d "$SCALA_SCALA_DIR" ]; then
+  echo "\$SCALA_SCALA_DIR must be set"
+  exit 1
+fi
+
+if [ ! -d "$DOTTY_DIR" ]; then
+  echo "\$DOTTY_DIR must be set"
+  exit 1
+fi
+
+Color_Off='\033[0m'
+Red='\033[0;31m'
+Green='\033[0;32m'
+failed=0
+
+run_tree_sitter () {
+  local source_dir=$1
+  local expected=$2
+  echo "Parse $source_dir:"
+  out=$(tree-sitter parse "$source_dir"/**/*.scala --quiet --stat || true)
+  actual=$(echo $out | rev | cut -d' ' -f1 | rev | sed 's/.$//')
+  echo "$out"
+  if (( $(echo "$actual > $expected" |bc -l) )); then
+    echo -e "  - ${Green}${actual}% ok${Color_Off}"
```

Comment:
Absolutely unnecessary, but would be nice to have: we could emit this green/red warning as a Github PR annotation: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#example-creating-an-annotation-for-an-error

Basically doing

```bash
echo "::error file=grammar.js,line=1::Missing semicolon"
```

Will show a message on the PR (similar to how coverage tools do it).

Reply:
Done

Reply:
Nice, I can see the correct messages in the actions log.

Your #82 will be the first one to demonstrate this working, as this PR doesn't touch grammar.js file so I guess the message just gets ignored.